edu.cuny.hunter.log.feature/feature.xml

```diff
@@ -293,19 +293,19 @@ version(s), and exceptions or additional permissions here}.&quot;
    </license>
 
    <url>
-     	<update label="Logging Level Evolution Update Site" url="https://raw.githubusercontent.com/ponder-lab/Logging-Level-Evolution-Plugin/master/edu.cuny.hunter.log.updatesite"/>
+      <update label="Logging Level Evolution Update Site" url="https://raw.githubusercontent.com/ponder-lab/Logging-Level-Evolution-Plugin/master/edu.cuny.hunter.log.updatesite"/>
       <discovery label="Common Eclipse Java Refactoring Framework" url="https://raw.githubusercontent.com/ponder-lab/Common-Eclipse-Java-Refactoring-Framework/master/edu.cuny.citytech.refactoring.common.updatesite"/>
    </url>
 
    <plugin
-         id="edu.cuny.hunter.mylyngit.core"
+         id="edu.cuny.citytech.refactoring.common.core"
```

Comment:
This doesn't look right at all.

Reply:
I removed and then added the plugin back again so that eclipse can register the version update. All the plugins are still the same